### ISSUE TITLE
Class 'App\User' not found

### ISSUE THREAD

**Issue by bhavinjr** [Laravel-Backpack/Base#287]
it show Class 'App\User' not found in backpack/base/src/app/Models/BackpackUser.php on line 8

My User model name space has been different how can i fix it? 
My Model name space is : **laravel\Models\User**..

**Comment by AbbyJanke** [comment]
Hello @bhavinjr, backpack uses the default 'web' guard for authentication which defines the model to be used as `App\User::class` if you update your `config/auth.php` file to have the correct path name it should work correctly:

based on your provided namespace it should look like this.
```php
'providers' => [
    'users' => [
        'driver' => 'eloquent',
        'model' => laravel\Model\User::class,
    ],
],
```

**Comment by tabacitu** [comment]
Excellent answer @AbbyJanke .

**Comment by bhavinjr** [comment]
Hello @AbbyJanke  my `config/auth.php` providers define properly

i guess you didn't understood my problem

open ` backpack/base/src/app/Models/BackpackUser.php` file 

and check the code
```
use App\User;

class BackpackUser extends User
{
```
`BackpackUser class extend to User (App\User)`

but my laravel project User Model namespace is `laravel\Models\User`

That's why its show  `Class 'App\User' not found` 

how can i fix without changing in package?

**Comment by lloy0076** [comment]
TL;DR You can't.

The quickest fix would be to copy the functions:

```
use Backpack\Base\app\Notifications\ResetPasswordNotification as ResetPasswordNotification;

// More code, e.g.
//
// class YourUser {
    /**
     * Send the password reset notification.
     *
     * @param string $token
     *
     * @return void
     */
    public function sendPasswordResetNotification($token)
    {
        $this->notify(new ResetPasswordNotification($token));
    }
    /**
     * Build the mail representation of the notification.
     *
     * @return \Illuminate\Notifications\Messages\MailMessage
     */
    public function toMail($notifiable)
    {
        return (new MailMessage())
            ->line([
                trans('backpack.base.password_reset.line_1'),
                trans('backpack.base.password_reset.line_2'),
            ])
            ->action(trans('backpack.base.password_reset.button'), url(config('backpack.base.route_prefix').'/password/reset', $this->token))
            ->line(trans('backpack.base.password_reset.notice'));
    }
```

And that might work depending on what you're trying to achive.

**Comment by vortgo** [comment]
@bhavinjr you should also edit config file `config/backpack/base.php`
`'user_model_fqn' => \App\Models\User::class,`
